docs/AboutUs.md

```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Ang Kuang Long
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/kuanglong.png" width="200px">
```

Comment:
The photo should be named after your GH username (see https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme)

Reply:
Done